sky/core.py

```diff
@@ -417,6 +405,10 @@ def queue(cluster_name: str,
         RuntimeError: if failed to get the job queue.
         sky.exceptions.ClusterNotUpError: the cluster is not up.
         sky.exceptions.NotSupportedError: the feature is not supported.
+        sky.exceptions.ClusterOwnerIdentityMismatchError: if the current user is
```

Comment:
Nit: can we copy paste the Raises section to other callers of check_cluster_available()?

Reply:
Done. Thanks!